File: BasicInputOutput/Java/SwapNumbers.java

import java.util.Scanner;

class  SwapNumbers{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter Two Value : ");
        int a = sc.nextInt();
        int b = sc.nextInt();

        System.out.println("Before Swap : "+a+", "+b);
        // with third variable ------------
            // int temp = a;
            // a = b;
            // b = temp;
        
        // Wihtout third variable
            a = a+b;
            b = a-b;
            a = a-b;
        System.out.println("After Swap : "+a+", "+b);
    }
}

File: BasicInputOutput/C/SwapNumbers.c
#include <stdio.h>

int main() {
    int a, b;
    printf("Enter Two Values: ");
    scanf("%d %d", &a, &b);

    printf("Before Swap: %d, %d\n", a, b);

    // with third variable
        // int temp  = a;
        // a=b;
        // b= temp;

    // Without third variable
    a = a + b;
    b = a - b;
    a = a - b;

    printf("After Swap: %d, %d\n", a, b);

    return 0;
}
